### DISCUSSION TITLE
调试代码时？如何使用配置文件进行调试

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#884] [Q&A] [answered]
发现调试只能按F6,正常应该有个填写配置文件的地方来进行调试，想问下，这个怎么搞？

**Comment by Jint-lzxy** [comment, marked as answer]
[`:h dap.txt`](https://github.com/mfussenegger/nvim-dap/blob/master/doc/dap.txt)

**Reply by wxzvim** [reply]
看帮助文档应该是:help dap.txt

**Reply by charliie-dev** [reply]
they r the same...

**Reply by wxzvim** [reply]
什么意思?

**Reply by wxzvim** [reply]
应该是:help dap才行

**Reply by Jint-lzxy** [reply]
Please always try to figure out the answer yourself before asking a question. Neovim has a comprehensive documentation for this purpose.
```console
[usr_20.txt] 20.2 Command line abbreviations
Some of the ":" commands are really long.  We already mentioned that
":substitute" can be abbreviated to ":s".  This is a generic mechanism, all
":" commands can be abbreviated.
```
We strongly recommend that you read the user manual from start to end: [usr_01.txt](https://neovim.io/doc/user/usr_01.html#usr_01.txt).

**Reply by wxzvim** [reply]
![image](https://github.com/ayamir/nvimdots/assets/110797126/9fc4fe6f-79a3-442d-9751-dc0f74004c12)
这命令没有找到相关文档说明哈

**Reply by wxzvim** [reply]
这个我知道，，但是不是说使用:h dap.txt可以查看dap相关插件帮助，，但是没有找到

**Reply by Jint-lzxy** [reply]
This plugin needs to be loaded first

**Reply by wxzvim** [reply]
哦哦，明白了

**Comment by wxzvim** [comment]
想问下，如何在vim里看插件的帮助文档？
